drivers/input/input_ft5336.c

```diff
@@ -88,24 +93,39 @@ static int ft5336_process(const struct device *dev)
 	}
 
 	points = (points >> TOUCH_POINTS_POS) & TOUCH_POINTS_MSK;
-	if (points != 0U && points != 1U) {
-		return 0;
-	}
-
-	/* obtain first point X, Y coordinates and event from:
-	 * REG_P1_XH, REG_P1_XL, REG_P1_YH, REG_P1_YL.
-	 */
-	r = i2c_burst_read_dt(&config->bus, REG_P1_XH, coords, sizeof(coords));
-	if (r < 0) {
-		return r;
-	}
+	if (points != 0) {
+		/* Any number of touches still counts as one touch.
+		 * All touch points except the first are ignored.
+		 * Obtain first point X, Y coordinates from:
+		 * REG_P1_XH, REG_P1_XL, REG_P1_YH, REG_P1_YL.
+		 * We ignore the Event Flag because Zephyr only
+		 * cares about pressed / not pressed and not
+		 * press down / lift up
+		 */
+		r = i2c_burst_read_dt(&config->bus, REG_P1_XH, coords, sizeof(coords));
+		if (r < 0) {
+			return r;
+		}
 
-	event = (coords[0] >> EVENT_POS) & EVENT_MSK;
-	row = ((coords[0] & POSITION_H_MSK) << 8U) | coords[1];
-	col = ((coords[2] & POSITION_H_MSK) << 8U) | coords[3];
-	pressed = (event == EVENT_PRESS_DOWN) || (event == EVENT_CONTACT);
+		row = ((coords[0] & POSITION_H_MSK) << 8U) | coords[1];
+		col = ((coords[2] & POSITION_H_MSK) << 8U) | coords[3];
 
-	LOG_DBG("event: %d, row: %d, col: %d", event, row, col);
+		/* Check that TOUCH_ID does not indicate an invalid touch.
+		 * If it does, we'll assume the touch event is over.
+		 */
```

Comment:
I think that the meaning is obvious enough from the code if you change it to suggestion below, I would drop this, good code is better than tangled code and comments.